src/status_im/ui/screens/signing/views.cljs

```diff
@@ -239,14 +243,26 @@
               {:type  :small
                :title :t/send-request-amount
                :error amount-error
-               :accessories [[acc-text (if amount (str amount) "0")
-                              (or display-symbol fee-display-symbol)]]}]
-             [separator]])
-          [list-item/list-item
-           {:type        :small
-            :title       :t/network-fee
-            :error       gas-error
-            :accessories [[acc-text fee fee-display-symbol] :chevron]
+               :accessories [react/nested-text {:style {:color colors/gray}} 
+                            [{:style {:color colors/black}} (if amount (str amount) "0")]
```

Comment:
CI won't accept this indent, run:
`make shell` then `lein cljfmt fix' to fix it

Reply:
Will do!